docs/run/nodes/nodes.md

```diff
@@ -16,9 +16,7 @@ At this time, any application that wishes to deploy to `mainnet` needs to run th
 
 **Ceramic networks** – There are currently three Ceramic networks: `mainnet`, `testnet-clay`, and `dev-unstable`. Learn more about each network [here](https://developers.ceramic.network/learn/networks/). By default, Ceramic will connect to `testnet-clay` and a [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service) running on Ethereum Ropsten. When you are ready to get on Ceramic `mainnet`, reach out on the [Ceramic Discord →](https://chat.ceramic.network) to get access to our `mainnet` anchor service.
 
-**Running IPFS** – Ceramic relies on a system called [IPFS](https://docs.ipfs.io/) to connect to and share data in Ceramic networks. IPFS can run in a "bundled" mode alongside the Ceramic daemon or in "unbundled" mode, in which Ceramic connects to IPFS over HTTP. **Bundled** mode is designed for testing and local development only. **Unbundled** mode is designed for production and accommodates both local and cloud setups. It allows for more configuration options, controlled resource allocation, maintenance, debugging, and observability. (Note that Ceramic only works with `go-ipfs` version 0.11 or later.)
-
-The rest of this guide assumes you are running Ceramic with IPFS unbundled.
```

Comment:
This assumption is still true based on how it is written. I don't think it makes sense to have a guide about running in production that includes how to run in bundled mode.

Reply:
put back but reworded from "unbundled" to "remote" to match what it's called in the config file